packages/camera/camera/example/pubspec.yaml

```diff
@@ -8,12 +8,9 @@ environment:
 
 dependencies:
   camera:
-    # When depending on this package from a real application you should use:
-    #   camera: ^x.y.z
-    # See https://dart.dev/tools/pub/dependencies#version-constraints
-    # The example app is bundled with the plugin so we use a path dependency on
-    # the parent directory to use the current plugin's version.
     path: ../
+  camera_platform_interface:
```

Comment:
This can be removed due to the comment about exporting `MediaSettings`.

Reply:
Removed.